runtime/client_http_request.go

```diff
@@ -56,13 +56,20 @@ func NewClientHTTPRequest(
 	clientID, methodName string,
 	client *HTTPClient,
 ) *ClientHTTPRequest {
+	scope := client.Scope.Tagged(map[string]string{
+		"client": clientID,
+		"method": methodName,
+	})
+
+	scopeFields := GetScopeFieldsFromCtx(ctx)
```

Comment:
It is not directly related to what this pr does, but we should refactor the function/variable names from `scopeFields` to `scopeTags` since `tags` is the standard terminology that tally uses. We can do it in a separate PR tho.